lms/static/js/student_account/views/AccessView.js

```diff
@@ -190,7 +191,7 @@ var edx = edx || {};
             this.element.scrollTop( $anchor );
 
             // Update url without reloading page
-            History.pushState( null, document.title, '/account/' + type + '/' );
+            History.pushState( null, document.title, '/account/' + type + '/' + queryParams );
```

Comment:
I don't think the slash at the end hurts, but originally I don't think we had one between the URL and the query params. Not necessarily important for this PR though.